ext/vscode/src/utils/azureDevCli.ts

```diff
@@ -126,13 +126,19 @@ function normalize(env: NodeJS.ProcessEnv): Environment {
 }
 
 function azdNotInstalledMsg(): string {
-    return localize("azure-dev.utils.azd.notInstalled", "Azure Developer CLI is not installed. Visit {0} to get it.", AzdInstallationUrl);
+    return localize("azure-dev.utils.azd.notInstalled", "Azure Developer CLI is not installed. Would you like to install it?.");
```

Comment:
The prompt comes up when the extension is activated, even when I activated it *to do the install*. We should suppress the prompt for the remainder of the session if they click "Yes" to install.